resources/admin.rb

```diff
@@ -0,0 +1,27 @@
+property :username, String, desired_state: false, identity: true, default: 'admin'.freeze
```

Comment:
Why creating a resource admin instead of a resource user?

Reply:
There will be a user resource to manage users (create, update, delete). I wanted to make the password change separate as they are not quite the same actions and it seemed weird to change a user password along with other attributes (email, etc.)